src/main/java/com/revature/controllers/PersonController.java

```diff
@@ -0,0 +1,56 @@
+package com.revature.controllers;
+
+import java.util.ArrayList;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.domain.Person;
+import com.revature.services.PersonLogic;
+
+@RestController
+@RequestMapping(value = "/api/v1/")
+public class PersonController {
+	
+	@Autowired
+	private PersonLogic personLogic;
+	
+	@RequestMapping(method = RequestMethod.GET, value = "persons/{id}")
+	public ResponseEntity<Person> getPerson(@PathVariable("id") Integer id) {
+		
+		return ResponseEntity.ok(personLogic.getPersonById(id));
+	}
+	
+	@RequestMapping(method = RequestMethod.GET, value = "trainees/")
+	public ArrayList<Person> getTrainees() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1319)